.travis.yml

```diff
@@ -7,7 +7,7 @@ services:
 language: go
 
 go:
-  - 1.8.x
+  - 1.9.x
```

Comment:
Should we run both until we drop 1.8.x support? I don't foresee as many lint/vet headaches as we had before with running multiple versions, maybe that is foolish to think.

Reply:
i'm fine going straight to 1.9.

people have to stop living in the past on old versions, plus it just makes tests twice as slow

Reply:
does this mean, the all consumer of ~d will have to upgrade? i'm trying to understand how are we thinking about upgrade process?

Reply:
Until the codebase uses some feature of Go 1.9 that is incompatible, I don't think this has any impact on consumers. Even once the server-side codebase might rely on something requiring post-Go 1.8.x, as long as the client library or client gRPC stubs are still compatible with older Go versions, I think consumers are still able to choose a Go version.

In summary, my opinion is that this PR is changing the intended support of Go versions for building containerd, but has no impact yet on consumers choice of Go version.

Reply:
Well, at the moment there's no formal release ;-).

But I suppose we should had decide what to do regarding the go version used once we reach GA.

I'm personally in favor of using the latest version, it comes with bug fixes and performance improvement, there's no real reason to stick with an old one.

Let's wait for @stevvooe to be back to argue on this